lib/guardian/db/adapter/ets.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Guardian.DB.ETSAdapter do
+  @moduledoc """
+  Implement the Guardian.DB.Adapter for ETS
```

Comment:
If everyone is :+1: on adding this adapter I'll expand on the module doc to talk about creating the table etc.

Reply:
👍as for me